README.md

```diff
@@ -37,13 +37,16 @@ Unlike a typical role, you need to configure [Ansible] to use an alternative [Py
 [coreos]
 host-01
 host-02
+```
 
+The role sets defaults for these, but you can set them yourself just in case:
+```ini
 [coreos:vars]
 ansible_ssh_user=core
-ansible_python_interpreter=/home/core/bin/python
+ansible_python_interpreter=/opt/python/bin/python
```

Comment:
@Adrian-Turjak, good idea use `/opt/` instead `$HOME`, but in this case you need to use flag `--become` because `core` user doen't have write permision to `/opt` directory. Documentation should be updated

Reply:
Done, added to playbook example